index.html

```diff
@@ -231,8 +231,66 @@ <h2>Open Source Village</h2>
 
     </div>
     </div>
+  </footer> -->
+  <footer class="footer" data-aos="fade-up" data-aos-duration="1500">
+    <div class="container">
+      <div class="footer_wrapper">
+        <div class="footer_box">
+          <div class="logo">
+            <div class="logo_img">
+              
+            </div>
+            <h2 class="fitbody">
+              <p>Open Source <strong>VILLAGER</strong> </p>
+            </h2>
+          </div>
+          <p class="abc" style="color: whitesmoke;">
+Join the Community now and become a Villager.
+          </p>
+        </div>
+        <div class="footer_box">
+          <h4 class="footer_title" style="color: whitesmoke;">Company</h4>
+          <ul class="footer_links">
+            <a href="/#">

```

Comment:
Could you please remove these links which have no work? or refer to #

Reply:
Sure also if you want I can link these to the profile pages of the project if any

Reply:
@Its-Aman-Yadav
##   please review and merge it now